.github/workflows/hourly.yaml

```diff
@@ -22,25 +22,26 @@ jobs:
     - name: Install dependencies for funcx-sdk and test requirements
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install ./funcx_sdk
-        python -m pip install -r funcx_sdk/test-requirements.txt
-    - name: Check for vulnerabilities in libraries
-      run: |
-        pip install safety
-        safety check
+        python -m pip install './funcx_sdk[test]'
+        python -m pip install safety
```

Comment:
i feel like this should be included as part of an `install[target]`

Reply:
I agree, it should be in the dev extra. I'll get it added.